doc/material-migration.md

```diff
@@ -1,4 +1,332 @@
-﻿# Updating to Uno.Material v2
+﻿---
+uid: uno.themes.material.migration
+---
+
+# Updating to Uno.Material v3
+
+Version 3.x of Uno.Material introduces support for [Lightweight Styling](lightweight-styling.md) as well as some breaking changes to the default style keys for some controls. The tables below provide details on the changes that have been made to the resource keys within Uno.Material.
```

Comment:
Should we add an additional sentence regarding the fact that most resources are now placed in theme dictionaries because of that so need to use "ThemeResource" instead of "StaticResource" when using them ?